packages/bruno-app/src/utils/codegenerator/auth.js

```diff
@@ -0,0 +1,29 @@
+import get from 'lodash/get';
+
+export const getAuthHeaders = (collectionRootAuth, requestAuth) => {
+  const auth = collectionRootAuth && ['inherit', 'none'].includes(requestAuth.mode) ? collectionRootAuth : requestAuth;
+
+  if (auth.mode === 'basic') {
```

Comment:
nit: `switch` may be cleaner here. Similar to `packages/bruno-app/src/utils/codegenerator/har.js`

Reply:
done